components/pages/toast.html

```diff
@@ -0,0 +1,92 @@
+            style="width: 820px; height: 474px;"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <p class="description">javascript code is added to open and close toast functionality. Used setTimeout for toast
+            to close after 3 seconds</p>
+        <iframe class="iframe-box"
+            src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=const%2520topToast%2520%253D%2520document.querySelector%28%2522%2523top-toast-id%2522%29%253B%250Aconst%2520topToastContent%2520%253D%2520document.querySelector%28%27%2523top-content%27%29%253B%250Aconst%2520topToastClose%2520%253D%2520document.querySelector%28%27%2523top-toast-close%27%29%253B%250A%250Aconst%2520bottomToast%2520%253D%2520document.querySelector%28%27%2523bottom-toast-id%27%29%253B%250Aconst%2520bottomToastContent%2520%253D%2520document.querySelector%28%27%2523bottom-content%27%29%253B%250Aconst%2520bottomToastClose%2520%253D%2520document.querySelector%28%27%2523bottom-toast-close%27%29%253B%250A%250Aconst%2520openTopToast%2520%253D%2520%28%29%2520%253D%253E%2520%257B%250A%2520%2520%2520%2520topToastContent.style.display%2520%253D%2520%27block%27%253B%250A%2520%2520%2520%2520setTimeout%28%28%29%2520%253D%253E%2520closeTopToast%28%29%252C%25203000%29%253B%250A%257D%250Aconst%2520closeTopToast%2520%253D%2520%28%29%2520%253D%253E%2520topToastContent.style.display%2520%253D%2520%27none%27%253B%250A%250AtopToast.addEventListener%28%2522click%2522%252C%2520openTopToast%29%253B%250AtopToastClose.addEventListener%28%2522click%2522%252C%2520closeTopToast%29%253B%250A%250A%250Aconst%2520openBottomToast%2520%253D%2520%28%29%2520%253D%253E%2520%257B%250A%2520%2520%2520%2520bottomToastContent.style.display%2520%253D%2520%27block%27%253B%250A%2520%2520%2520%2520setTimeout%28%28%29%2520%253D%253E%2520closeBottomToast%28%29%252C%25203000%29%253B%250A%257D%250Aconst%2520closeBottomToast%2520%253D%2520%28%29%2520%253D%253E%2520bottomToastContent.style.display%2520%253D%2520%27none%27%253B%250A%250AbottomToast.addEventListener%28%2522click%2522%252C%2520openBottomToast%29%253B%250AbottomToastClose.addEventListener%28%2522click%2522%252C%2520closeBottomToast%29%253B"
+            style="width: 828px; height: 654px;" sandbox="allow-scripts allow-same-origin">
+        </iframe>
+    </div>
+    <script src="../javascript/toast.js" type="text/javascript"></script>
```

Comment:
Suggestion - you can work on this later. 

But instead of separating out the javascript for the component outside the pages folder. You could create a subfolder within components called `toast`. And within that subfolder add thee HTML, JavaScript, CSS.

Reply:
Updated the toast folder structure will also update other components also,   by creating a branch. Thank you for the suggestion